src/js/chartAuthoring.js

```diff
@@ -144,4 +147,18 @@ https://raw.githubusercontent.com/fluid-project/chartAuthoring/master/LICENSE.tx
         });
         return pieChartData;
     };
+
+    // Adds aria attributes that only make sense within the context of the overall chart authoring tool
+    // Specifically, adds:
+    // - an aria-controls attribute for the dataEntryPanel's form referencing the unique IDs of the pie and legend
+    floe.chartAuthoring.addAriaConnections = function(that) {
+
+        var legendId = floe.d3ViewComponent.getOrCreateId(that.chartAuthoringInterface.pieChart.legend.locate("table")),
```

Comment:
Sorry for providing an incomplete review comment last time. Infusion has a built-in function fluid.allocateSimpleId() that you can use for this purpose. Thanks.